### ISSUE TITLE
pgtyped/cli v2.2.0 generate wrong path folder on windows

### ISSUE THREAD

**Issue by kwatanabe1050** [adelsz/pgtyped#519]
**Describe the bug**
v2.2.0 cli uses wrong path separator on windows like this: `src&#92;controllers&#92;system&#92;health\getHealth.queries.ts`
It creates the folder including `&#92;`.

**Expected behavior**
Create correct path folder as same as v2.1.0 cli.
This is happened after update v2.2.0.

**Test case**
My environment is `typescript` v5.0.4, `@pgtyped/runtime` v2.1.0 `@pgtyped/cli` v2.2.0  as latest on node v18.15.0.

```powershell
> pgtyped -w -c pgTyped.config.json
Saved 1 query types from src\controllers\system\health\getHealth.sql to src&#92;controllers&#92;system&#92;health\getHealth.queries.ts
```

```json
{
  "transforms": [
    {
      "mode": "sql",
      "include": "**/*.sql",
      "emitTemplate": "{{dir}}/{{name}}.queries.ts"
    },
  "srcDir": "./src/",
  "failOnError": false,
  "camelCaseColumnNames": true,
  ],
"db": {...}
}
```

I used v.2.1.0 package with typescript v5 with`npm i -f` option to ignore engines error.
So only cli package version is changed from normal case project.

Thank you.

**Comment by kwatanabe1050** [comment]
I feel only 1 line change will work but I don't know how to create build/test package and send MR.

![image](https://user-images.githubusercontent.com/48498820/235570873-d1793d83-717d-4571-8c35-572b17a6ee7b.png)

**Comment by adelsz** [comment]
This should be fixed in v2.2.1.
Unfortunately I don't have a Windows machine to test and our testing pipeline relies on Docker which is unsupported on Windows.
If the issue persists, please reopen.

**Comment by kwatanabe1050** [comment]
I tried cli v2.2.1 on my local windows and it works.

Thanks for quick support!
